ext/sodium/sodium_pwhash.c

```diff
@@ -183,19 +172,13 @@ PHP_MINIT_FUNCTION(sodium_password_hash) /* {{{ */ {
 	if (FAILURE == php_password_algo_register("argon2i", &sodium_algo_argon2i)) {
 		return FAILURE;
 	}
-	REGISTER_STRING_CONSTANT("PASSWORD_ARGON2I", "argon2i", CONST_PERSISTENT);
+	register_sodium_pwhash_symbols(module_number);
 
 	if (FAILURE == php_password_algo_register("argon2id", &sodium_algo_argon2id)) {
 		return FAILURE;
 	}
 	REGISTER_STRING_CONSTANT("PASSWORD_ARGON2ID", "argon2id", CONST_PERSISTENT);
```

Comment:
Why did you kept this one here? It seems to me that this could also be added to the stub file

Reply:
You're right and your explanation in your other comment was very helpful in helping me see how this should be done.